addons/sale_pdf_quote_builder/tests/test_pdf_quote_builder.py

```diff
@@ -20,25 +21,46 @@ def setUpClass(cls):
 
     def test_dynamic_fields_mapping(self):
         sol_1, sol_2 = self.sale_order.order_line
-        fields_to_map = {
+        fields_to_map_ok = {
             f'sol_id_{sol_1.id}__price_unit',  # float
             f'sol_id_{sol_2.id}__user_id__name',  # char
             f'sol_id_{sol_2.id}__validity_date',  # date
             f'sol_id_{sol_1.id}__delivery_date',  # datetime
             f'sol_id_{sol_1.id}__tax_excl_price',  # datetime
-
+        }
+        fields_to_map_ko = {
             # TODO
             f'sol_id_{sol_2.id}__user_id',  # relational
             f'sol_id_{sol_2.id}__state',  # selection
             f'sol_id_{sol_1.id}__is_downpayment',  # boolean
             f'sol_id_{sol_2.id}__all_products',  # relational multi
         }
+        fields_to_map = fields_to_map_ok | fields_to_map_ko
```

Comment:
```suggestion
        fields_to_map = fields_to_map_ok | fields_to_map_ok
```

Reply:
not sure why you're suggesting doing a union of a set with itself?

Reply:
Because I read too fast, nevermind >.<